se/Mortgage/v0/schema/MortgageMessage.yaml

```diff
@@ -0,0 +1,37 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/MortgageMessage"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: MortgageMessage
```

Comment:
Yeah, like a comment with conditions/terms/info for the offer (we do that in SE now as well)?

Reply:
Yes indeed, an that would then be tied to the offer

Reply:
Perhaps as two fields, one for customer facing info and one for non-customer faacing

Reply:
I added property `comment` to MortgageOffering (just like in PrivateUnsecuredLoan.SE). I think having two different fields may complicate things rather than improve them. What info that is suitable for the customer or (solely) the broker administrator should probably be left up to the broker to decide.